_collections/_news/2024-10-30-november-2024-upcoming-events.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+published: true
+title: November 2024 - Upcoming Events
+thumbnail: /assets/images/2024-upcoming-events/halloween-quiz.png
+header-image: /assets/images/2024-upcoming-events/cream-tea.png
+---
+Hello everyone! We hope you've been able to settle well into university.
+
+We've had an incredible start to the semester with our Cream Tea and Halloween Quiz events, and have got to meet so many
+of you. For our first session of Missing Semester, we had to book three(!!) rooms to have everyone join the event, and
+we had to order 40 pizzas from Papa John's for our CS Societies Night, only to be told off for ordering too much. 
+However, none of this would've happened if it wasn't for all of you. Thank you for making these events unforgettable,
+as CSS would not be the same without you.
+
+This month, we've got a big collection of events coming up, but this list isn't exhaustive - expect some surprises
+around the corner!
+
+## Lua Workshops 🛠️ - Thursday 31st October, 5pm - Y3-G29
+This Thursday, we have Amrit giving the second Lua workshop that covers the following:
+- The Lua C API and embedding Lua
+- Making modules for Lua
+- Setting up Lua in your editor
+- Lua package management
+- Type safety
+
+If you missed the first talk, you can access the "slides" for it here: https://www.frityet.net/lua-ss/ (of which we were
```

Comment:
```suggestion
If you missed the first talk, you can access the "slides" for it here: <https://www.frityet.net/lua-ss/> (of which we were
```

Reply:
What do these generally do? I presume open the link in a new tab?

Reply:
Turns them into actual hyperlinks:

Before
![image](https://github.com/user-attachments/assets/bad40008-60c7-43ec-9146-27b92d9598bb)

After
![image](https://github.com/user-attachments/assets/146db1a4-6d07-48cf-8108-ccde575351de)